code/modules/customitems/item_defines.dm

```diff
@@ -313,6 +313,11 @@
 	icon_state = "eyepro"
 	item_state = "eyepro"
 
+/obj/item/clothing/glasses/hud/security/sunglasses/fluff/voxyHUD //LP Spartan: Kaskreyarawkta
```

Comment:
AFAIK it's just keeping with the general style of how things are written here, as we can see that every parent of voxyHUD all the way down the line is lowercase

Totally appearance-related